students/thorn/lesson04/trigram.py

```diff
@@ -34,10 +34,13 @@ def read_text(intext):
 def create_trigrams_try_two(text_list):
     trigrams = {}
     # Add 3rd word to trigram pairs.  Create key entry if not already in dict.
+    # TODO: learn about default dicts :(
```

Comment:
`defaultdict` is awesome!

here is a quick demonstration:
```
from collections import defaultdict

dirs = defaultdict(list)
dirs["directory1"].append("file1")
dirs["directory1"].append("file2")
print(dirs)
print(dict(dirs))
```

the only difference is how you declare your dictionary and then you just use without worrying about getting a KeyError on append with a new key for example